eng/WpfArcadeSdk/tools/RuntimeFrameworkReference.targets

```diff
@@ -25,7 +29,7 @@
   </ItemGroup>
 
   <PropertyGroup>
-    <!-- If TargetFramework is not netcoreapp3.x, then reset RuntimeFrameworkVersion -->
-    <RuntimeFrameworkVersion Condition="!$(TargetFramework.StartsWith('netcoreapp3.'))" />
+    <!-- If TargetFramework is not one of the supported .NET Core versions, then reset RuntimeFrameworkVersion -->
```

Comment:
Is the core moniker applicable for 5.0?  I don't care so much about the comments, but are there error/warning messages specific to WPF we need to update?  (Though those are somewhere else, right?)

Reply:
Error messages like what? The ones in WindowsDesktop SDK or NET SDK rely on `<` or `<=` tests - so no need to update them.  Everywhere else that needed updates to `netcoreapp3.0` related props/targets/*proj have been updated. 

.NET 5.0 is `netcoreapp5.0` - so yeah, I would expect it to have `TargetFrameworkIdentifier==.NETCoreApp`. 

See [this](https://github.com/dotnet/sdk/blob/fe6b50c78426124e34e56ecb7df8b6b86e6e0b48/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.TargetFrameworkInference.targets#L52-L57) in https://github.com/dotnet/sdk: 
```xml
  <!-- Map short name to long name. See earlier comment for example of how to work with identifiers that are not recognized here. -->
  <PropertyGroup Condition="'$(TargetFrameworkIdentifier)' == ''">
    <TargetFrameworkIdentifier Condition="'$(_ShortFrameworkIdentifier)' == 'netstandard'">.NETStandard</TargetFrameworkIdentifier>
    <TargetFrameworkIdentifier Condition="'$(_ShortFrameworkIdentifier)' == 'netcoreapp'">.NETCoreApp</TargetFrameworkIdentifier>
    <TargetFrameworkIdentifier Condition="'$(_ShortFrameworkIdentifier)' == 'net'">.NETFramework</TargetFrameworkIdentifier>
  </PropertyGroup>
```